content/en/docs/getting-started/hardware-requirements.md

```diff
@@ -61,28 +59,28 @@ You can build a [distributed cluster]({{% ref "/docs/operations/stretched/" %}})
 
 **Networking:**
 
-- Distributed cluster requires both a fast and reliable network **and** low RTT (Round Trip Time), because
+- Distributed cluster requires both fast and reliable network, and it **must** have low RTT (Round Trip Time), as
   Kubernetes is not designed to operate efficiently over high-latency networks.
 
-  Two datacenters in the same city typically have less than 1 ms latency, which is ideal.
+  Datacenters in the same city typically have less than 1 ms latency, which is ideal.
   The *maximum acceptable* RTT is 10 ms.
   Running Kubernetes or replicated storage over a network with RTT above 20 ms is strongly discouraged.
-  Use the `ping` command between locations to measure actual RTT.
+  To measure actual RTT you can use the `ping` command.
 
 - It's also recommended to have at least 2–3 nodes per datacenter in a distributed cluster.
-  This ensures that if one datacenter loses connectivity, the cluster can survive the outage without major disruption.
+  This ensures that the cluster would be able to survive one datacenter loss without major disruption.
 
-- If it's impossible to keep an internal network addressing scheme (such as VPN between datacenters),
-  you can enable **KubeSpan**—a Talos Linux feature that creates a WireGuard-backed full-mesh VPN between nodes.
+- If it's hard to keep a single address space between datacenters, instead of using some external VPN,
+  you can enable **KubeSpan**, a Talos Linux feature that creates a WireGuard-backed full-mesh VPN between nodes.
 
-## High-Availability applications
+## Highly-Available applications notes
```

Comment:
I agree, it looks like just notes or additions to the previous chapters.

Should we merge it to Production? It has a note on HA in the Compute section.